kyo-prelude/shared/src/main/scala/kyo2/package.scala

```diff
@@ -6,8 +6,12 @@ import kyo2.kernel.Safepoint
 private[kyo2] type Frame = kernel.Frame
 private[kyo2] inline def Frame = kernel.Frame
 
-export kernel.<
```

Comment:
I had to deal with a few cryptic compiler crashes while working on this change that went away once I avoided `export`s here.